cppapi/client/devapi_base.cpp

```diff
@@ -5544,6 +5548,7 @@ DevicePipe DeviceProxy::write_read_pipe(DevicePipe &pipe_data)
 // Prepare data sent to device
 //
 
+    //TODO replace with createDevPipeData
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this "TODO" comment. ([cpp:S1135](https://sonarcloud.io/organizations/tango-controls/rules#rule_key=cpp%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.tango-controls%3Acpp-tango%3Atango-9-lts&pullRequest=477&issues=AWWPm0Fu3Qu1svVwsabE&open=AWWPm0Fu3Qu1svVwsabE)

[ISSUE_KEY]: AWWPm0Fu3Qu1svVwsabE